<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoursesDialog</class>
 <widget class="QDialog" name="CoursesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>259</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>501</width>
    <height>259</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>501</width>
    <height>259</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menadżer kursów</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>488</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnAddCourse">
      <property name="toolTip">
       <string>Dodanie nowego kursu z pliku do bazy</string>
      </property>
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDelete">
      <property name="toolTip">
       <string>Usunięcie kursu z bazy</string>
      </property>
      <property name="text">
       <string>Usuń</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEdit">
      <property name="text">
       <string>Edytuj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCreateCourse">
      <property name="toolTip">
       <string>Stworzenie nowego kursu</string>
      </property>
      <property name="text">
       <string>Utwórz</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="toolTip">
       <string>Wybranie nowego kursu. UWAGA w przypadku kiedy kurs jest już wybrany to jest on zastępowany nowym</string>
      </property>
      <property name="text">
       <string>Wybierz</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="toolTip">
       <string>Powrót do poprzedniego okna</string>
      </property>
      <property name="text">
       <string>Zamknij</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>181</height>
    </rect>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="selectionRectVisible">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>CoursesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CoursesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
